examples/cookbook/forms/validation/analysis_options.yaml

```diff
@@ -1 +1,9 @@
-include: package:pedantic/analysis_options.yaml
\ No newline at end of file
+# Take our settings from the example directory's main analysis_options.yaml
+# file. If necessary for a particular example, this file can also include
+# overrides for individual lints.
+
+include: ../../../analysis_options.yaml
+
+linter:
+  rules:
+    prefer_const_literals_to_create_immutables: false
```

Comment:
I'm curious how many times this was triggered such that it was worth blanket elision instead of just commenting out the individual instances.